Clarified how `min-width` and `max-width` override `width` (#27152)
mdn/content: files/en-us/web/css/width/index.md

{{EmbedInteractiveExample("pages/css/width.html")}}

> The specified value of `width` applies to the content area so long as its value remains within the values defined by {{cssxref("min-width")}} and {{cssxref("max-width")}}.

Reviewer: ```suggestion
The specified value of `width` applies to the content area so long as its value remains within the values defined by {{cssxref("min-width")}} and {{cssxref("max-width")}}.

```

> The {{cssxref("min-width")}} and {{cssxref("max-width")}} properties override `width`. If `width` is greater than {{cssxref("max-width")}} or less than {{cssxref("min-width")}}, it will not apply to the [content area](/en-US/docs/Web/CSS/CSS_Box_Model/Introduction_to_the_CSS_box_model#content_area) because of these override properties.
> 
> However, `width` _will_ apply and resize the [content area](/en-US/docs/Web/CSS/CSS_Box_Model/Introduction_to_the_CSS_box_model#content_area) if its value remains within the boundaries defined by {{cssxref("min-width")}} and/or {{cssxref("max-width")}}.

Reviewer: ```suggestion
The specified value of `width` applies to the content area as long as the value is within the minimum and maximum width values defined by {{cssxref("min-width")}} and {{cssxref("max-width")}}, respectively. If the specified `width` is less than `min-width` and/or the the specified `width` is  greater than `max-width`, then the `min-width` and `max-width` values override the `width` value.
```

- removed the link to "content area" because it already exists in the first para on the page

Reviewer: Hello @Snowyamur, if you agree with the suggestion, you can commit them and we can land the changes 👍

Author: I think the suggestion is clearer and better than what I came up with @dipikabh. Though I think it could be made just a little bit clearer.

If you don't mind, I copied what you suggested and revised it a bit below, resolving any typos and trying to make it as concise as possible:

```suggestion
The specified value of `width` applies to the content area so long as its value remains within the values defined by {{cssxref("min-width")}} and {{cssxref("max-width")}}.
- If the value for 'width' is less than the value for 'min-width', then 'min-width' overrides 'width'.
- If the value for 'width' is greater than the value for 'max-width', then 'max-width' overrides 'width'.
```

Please let me know if you think this updated commit change will suffice. If so, you can go ahead and land the changes.

Reviewer: Looks good 👍 Thanks for the rewording. I updated the single quotes with the tick character.
